pkg/asset/installconfig/aws/validation.go

```diff
@@ -49,6 +53,13 @@ func Validate(ctx context.Context, meta *Metadata, config *types.InstallConfig)
 		return errors.New(field.Required(field.NewPath("platform", "aws"), "AWS validation requires an AWS platform configuration").Error())
 	}
 	allErrs = append(allErrs, validateAMI(ctx, config)...)
+	if err := checkForEIPAllocationsWithNLB(config); err != nil {
+		fmt.Printf("error %v", err)
+		return errors.New(field.Required(field.NewPath("platform", "aws", "lbType"), err.Error()).Error())
```

Comment:
It's best to return a formatted `allErrs` here, not just the one error, since we call `validateAMI` prior to this.

Reply:
done.

Reply:
@miheer there is commented code here still.